sts_auth/stsauth.py

```diff
@@ -93,39 +110,37 @@ def parse_config_file(self):
         """Read configuration file and only set values if they
         were not passed in from the CLI.
         """
-        if self.config.has_section('default'):
-            logger.debug('Found \'default\' section in'
-                         ' {0.credentialsfile!r}!'.format(self))
-            default = self.config['default']
-            msg = ('Attribute {1!r} not set, using value from {0.credentialsfile!r}')
+        if self.config.has_section("default"):
+            logger.debug("Found 'default' section in" " {0.credentialsfile!r}!".format(self))
+            default = self.config["default"]
+            msg = "Attribute {1!r} not set, using value from {0.credentialsfile!r}"
             if not self.region:
-                logger.debug(msg.format(self, 'region'))
-                self.region = default.get('region')
+                logger.debug(msg.format(self, "region"))
+                self.region = default.get("region")
             if not self.output:
-                logger.debug(msg.format(self, 'output'))
-                self.output = default.get('output')
+                logger.debug(msg.format(self, "output"))
+                self.output = default.get("output")
             if not self.idpentryurl:
-                logger.debug(msg.format(self, 'idpentryurl'))
-                self.idpentryurl = default.get('idpentryurl')
+                logger.debug(msg.format(self, "idpentryurl"))
+                self.idpentryurl = default.get("idpentryurl")
             if not self.domain:
-                logger.debug(msg.format(self, 'domain'))
-                self.domain = default.get('domain')
+                logger.debug(msg.format(self, "domain"))
+                self.domain = default.get("domain")
             if not self.okta_org:
-                logger.debug(msg.format(self, 'okta_org'))
-                self.okta_org = default.get('okta_org')
+                logger.debug(msg.format(self, "okta_org"))
+                self.okta_org = default.get("okta_org")
             if not self.okta_shared_secret:
-                logger.debug(msg.format(self, 'okta_shared_secret'))
-                self.okta_shared_secret = default.get('okta_shared_secret')
+                logger.debug(msg.format(self, "okta_shared_secret"))
+                self.okta_shared_secret = default.get("okta_shared_secret")
         else:
-            logger.debug('Could not find \'default\' section in'
-                         ' {0.credentialsfile!r}!'.format(self))
+            logger.debug("Could not find 'default' section in" " {0.credentialsfile!r}!".format(self))
```

Comment:
Changes like these should be merged into one string if they're moved to a single.

Reply:
@phillipjf fixed